src/Microsoft.TestPlatform.Common/Resources/Resources.resx

```diff
@@ -166,10 +166,10 @@
     <value>Error: Invalid operator '{0}'</value>
   </data>
   <data name="LoggerInitializationError" xml:space="preserve">
-    <value>Exception occurred while initializing logger with URI '{0}'.  The logger will not be used.  Exception: {1}</value>
+    <value>Exception occurred while initializing logger with {0}: '{1}'.  The logger will not be used.  Exception: {2}</value>
```

Comment:
Run build.cmd, then add the other lang changes as well.

Reply:
done